packages/labs/react/package.json

```diff
@@ -80,7 +81,7 @@
       ]
     },
     "build:ts:types": {
-      "command": "treemirror development . \"**/*.d.ts{,.map}\" \"!development/test\"",
+      "command": "treemirror development . \"**/*.d.ts{,.map}\" \"!test\"",
```

Comment:
Is this change expected?

Reply:
Yes intended! It was incorrect when I added this a while back. Since we already set `development` as the source dir in the first argument, the glob starts relative to that. It was no harm that it wasn't working, just unnecessarily copying extra test files.